lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,16 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
-
-      if (method === 'POST' && !kid) {
-        return;
-      }
-
+      // if (method === 'POST' && !kid) {
+      //   return;
+      // }
```

Comment:
ahh I see - thanks for explaining more! The `response` comes back as `null` - instead of `return` [here](https://github.com/DoSomething/phoenix/blob/b43bc156041a59dcb08894df4132ad93065be3e1/lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc#L210), should I set the `kid` to something like `0`?